pkgs/development/libraries/pipewire/alsa-profiles-use-libdir.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/meson.build b/meson.build
+index ffee41b4..f3e4ec74 100644
+--- a/meson.build
++++ b/meson.build
+@@ -53,7 +53,7 @@ endif
+ 
+ spa_plugindir = join_paths(pipewire_libdir, spa_name)
+ 
+-alsadatadir = join_paths(pipewire_datadir, 'alsa-card-profile', 'mixer')
++alsadatadir = join_paths(pipewire_libdir, '..', 'share', 'alsa-card-profile', 'mixer')
```

Comment:
This directory is actually used by the SPA plugin and not the compatibility layer. The reason it was moved to `pipewire_libdir` is that there is a direct dependency link to which was causing cyclic dependencies. It might make more since to move it to `$lib/lib/spa-0.2/alsa/mixer` to make it more obvious.

The compatibility layer directories are `$lib/lib/alsa-lib` and `$out/share/alsa`. These could be moved to a separate output but are quite small and wouldn't remove the dependency on `alsaLib` due to the previously mentioned SPA plugin. I am open to this if it is preferred though.

Reply:
I guess this is fine for now, thanks for the explanation.